Assets/MXR.SDK/Runtime/MXRManager.cs

```diff
@@ -22,39 +27,102 @@ public static class MXRManager {
         /// </summary>
         public static HomeScreenState HomeScreenState { get; private set; }
 
+        [Obsolete("This event was used with the Init method which is being deprecated. " +
+        "Use InitAsync instead.", false)]
         /// <summary>
         /// Fired when <see cref="MXRManager"/> is initialized
         /// </summary>
         public static event Action OnInitialize;
 
+        [Obsolete("This boolean property was used with the Init method which is being deprecated. " +
+        "Use InitAsync instead.", false)]
         /// <summary>
         /// Whether <see cref="MXRManager"/> has been initialized
         /// </summary>
         public static bool IsInitialized { get; private set; } = false;
 
+        /// <summary>
+        /// Auto-initializes <see cref="System"/> with appropriate implementation
+        /// based on the runtime environment and awaits until the system is available 
+        /// for the first time.
+        /// </summary>
+        /// <param name="system"></param>
+        /// <returns>The <see cref="IMXRSystem"/> instance in MXRManager</returns>
+        async public static Task<IMXRSystem> InitAsync(IMXRSystem system = null) {
+#pragma warning disable 0618
```

Comment:
Can you add a comment explaining why this is ok to disable and restore?

Reply:
Added!